.gitignore

```diff
@@ -48,3 +48,8 @@ checkouts
 # RVM files for localised setups
 .ruby-gemset
 .ruby-version
+
+# clj-i18n
+*.class
+/resources/locales.clj
+mp-*
```

Comment:
May not be relevant if the files might be scattered anywhere by i18n, but if we only expect the .class and mp- (or any) files in a particular scope (subdir, etc.), I'd be tempted to construe the .gitignore patterns as narrowly as possible, so that we still notice unexpected bits elsewhere.